torch/onnx/__init__.py

```diff
@@ -179,7 +179,11 @@ def forward(self, k, x):
             optimization is applied to the model during export. Constant-folding
             optimization will replace some of the ops that have all constant
             inputs, with pre-computed constant nodes.
-        example_outputs (tuple of Tensors, default None): Model's example outputs being exported.
+        example_outputs (tuple of Tensors, list of Tensors, Tensor, int, float, bool, default None):
+            Model's example outputs being exported. 'example_outputs' must be provided when exporting
+            a ScriptModule or TorchScript Function. If there is more than one item, it should be passed
+            in tuple format, e.g.: example_outputs = tuple([x, y, z]). Otherwise, only one item should
```

Comment:
would it be better to use `tuple(x, y, z)`?

Reply:
I don't this expression would work: tuple(x, y, z). But it represents the idea.

Reply:
you are right, should be `example_outputs = (x, y, z)`.